pkg/apic/apiserver/models/management/v1alpha1/model_dataplane_spec_git_hub.go

```diff
@@ -0,0 +1,18 @@
+/*
+ * API Server specification.
+ *
+ * No description provided (generated by Openapi Generator https://github.com/openapitools/openapi-generator)
+ *
+ * API version: SNAPSHOT
+ * Generated by: OpenAPI Generator (https://openapi-generator.tech)
+ */
+
+package management
+
+// DataplaneSpecGitHub The configuration common to all GitHub agents that use this dataplane
+type DataplaneSpecGitHub struct {
+	Type            string                            `json:"type,omitempty"`
+	RepositoryName  string                            `json:"repositoryName"`
+	RepositoryOwner string                            `json:"repositoryOwner"`
```

Comment:
I am not a fan of repeating the word repository for both these properties. Can we drop that in api-server and just make them name and owner?